routes/api/posts.js

```diff
@@ -0,0 +1,23 @@
+const router = require("express").Router();
+const postsController = require("../../controllers/postsController");
+const commentsController = require("../../controllers/commentsController");
+
+// Posts routes
+router
+  .route("/")
+  .get(postsController.findAll)
+  .post(postsController.create);
+
+router.route("/:id")
+  .get(postsController.findOne);
```

Comment:
Your posts controller doesn't have a `findOne` method on it. Did you mean for this to be `findById`?

Reply:
I realize AFTER looking in docs that findOne is a method specific to mongoose.  Will update.